src/pages/forget-password.tsx

```diff
@@ -11,22 +11,16 @@ import { useForgetPasswordMutation } from '@generated/graphql.queries';
 export default function ForgetPassword() {
   const { t } = useTranslation('auth');
   const head = { title: t('forget-password.title') };
-
-  const [successMessage, setSuccessMessage] = useState('');
   const { gqlClient } = useApiClient();
   const {
     mutate: forgetPasswordSendEmail,
     error,
     isLoading: isSendingEmail,
   } = useForgetPasswordMutation(gqlClient, {
-    onSettled: (data, error) => {
+    onSettled: (error) => {
       if (error) {
         // @ts-ignore
         console.error(error);
-        setSuccessMessage('');
-      }
-      if (data) {
-        setSuccessMessage(t('forget-password.success'));
```

Comment:
Maybe we can push the page to the success page when data is returned successfully
we can still use `router.push()` here